include/wx/image.h

```diff
@@ -628,6 +651,46 @@ class WXDLLIMPEXP_CORE wxImage: public wxObject
     // modified versions of this image.
     wxImage MakeEmptyClone(int flags = Clone_SameOrientation) const;
 
+    // Helper function used internally by wxImage class only.
+    template <typename T>
+    void ApplyToAllPixels(void (wxImage::*func)(unsigned char *, T), T value);
+
+    // Makes an RGB value disabled.
+    void DoMakeDisabled(unsigned char *rgb, unsigned char brightness);
+
+    // red, green and blue are weight doubles, they are used internally by
```

Comment:
Does "weight" mean "in [0..1] range"? If so, it'd be useful to mention this in the comment explicitly.

Reply:
Exactly.